shell/ev-toolbar.c

```diff
@@ -310,11 +332,12 @@ ev_toolbar_init (EvToolbar *ev_toolbar)
 }
 
 GtkWidget *
-ev_toolbar_new (EvWindow *window)
+ev_toolbar_new (EvWindow *window, GSettings *settings)
```

Comment:
Each function argument should be on it's own line. Look at the other functions and try to stay consistent.